### PR TITLE
Load results from proposals if available

### PR DESCRIPTION
**Pull request by bonustrack**
TODO:
- [ ] Store votes VP on db
- [ ] Load only top 10 votes then paginate
- [ ] Update scores when there is a new vote or scores is older than 30 seconds

### CODE REVIEW

**Review by mktcode — COMMENTED** [review]
When voting powers are updated in the db and included in the graphql response, can't we then remove the `loadResults` function entirely?

File: src/views/SpaceProposal.vue [hunk 1]
```diff
@@ -101,15 +102,28 @@ async function loadProposal() {
...
+  }
   proposalObj.value.votes = await getProposalVotes(id);
   const resultsObj = await getResults(
     props.space,
     proposalObj.value.proposal,
     proposalObj.value.votes
   );
-  results.value = resultsObj.results;
-  votes.value = resultsObj.votes;
+  if (proposal.value.id !== ens) results.value = resultsObj.results;
```
Comment (mktcode) [inline src/views/SpaceProposal.vue:123]: I don't get this one. The request is still made but results are just not used for the ENS proposal?

Comment (bonustrack) [reply src/views/SpaceProposal.vue:123]: The hub return empty array for votes on ENS proposal for now

**Comment by bonustrack** [comment]
@mktcode Thats the idea, but will need few changes

**Pull request merged**